### PR TITLE
SUBS-696 MG Menu Exp Mobile

### PR DESCRIPTION
**Pull request by eddieferrer**
* Adds monthly good menu exp to mobile
* Change menu exp button capitalization

### CODE REVIEW

File: src/components/WwwFrame/TheHeader.vue [hunk 1]
```diff
@@ -409,7 +411,7 @@ export default {
 		PromoBannerLarge,
 		PromoBannerSmall,
 		SearchBar,
-		TheLendMenu: () => import('@/components/WwwFrame/LendMenu/TheLendMenu'),
```
Comment (eddieferrer) [inline src/components/WwwFrame/TheHeader.vue:412]: This dynamic import was breaking the exp wrapper component, which has a duplicate slot with the lend menu

Comment (mcstover) [reply src/components/WwwFrame/TheHeader.vue:412]: So now that will get bundled into the header bundle instead of being an on demand import?

Comment (eddieferrer) [reply src/components/WwwFrame/TheHeader.vue:412]: I believe thats true, but I am not sure what repercussions that will have. @emuvente might know more?

Comment (emuvente) [reply src/components/WwwFrame/TheHeader.vue:412]: Yeah, that's all that should happen. We previously used that dynamic import to delay the fetching of lend menu data, but that's handled by kv-dropdown events now

**Review by emuvente — APPROVED** [review]

**Review by mcstover — APPROVED** [review]

**Pull request merged**